src/kv/idbstore.rs

```diff
@@ -294,7 +296,12 @@ impl Write for WriteTransaction {
         self.pending
             .lock()
             .await
-            .insert(key.to_string(), value.to_vec());
+            .insert(key.to_string(), Some(value.to_vec()));
```

Comment:
I thought we decided to do `key.into()`?

Reply:
Yeah, we did, thank you. I'll do a quick fixup pass across the codebase now.

Reply:
https://github.com/rocicorp/repc/commit/90c44f56a807bb6478aa60a992766b736265eb0e